nntrainer/layers/lstmcell_core.cpp

```diff
@@ -0,0 +1,519 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 hyeonseok lee <hs89.lee@samsung.com>
+ *
+ * @file   lstmcell_core.cpp
+ * @date   25 November 2021
+ * @brief  This is LSTMCellCore Layer Class of Neural Network
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author hyeonseok lee <hs89.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+
+#include <cmath>
+#include <lazy_tensor.h>
+#include <lstmcell_core.h>
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+#include <node_exporter.h>
+#include <util_func.h>
+
+// Todo: remove this if sharing wt_idx with other lstm variant is enabled
+#define ENABLE_SHARING_WT_IDX 0
+
+namespace nntrainer {
+
+void fillLayerInitContext(InitLayerContext &context,
```

Comment:
those functions (`fill*`) should be static or either wrapped with namespae {} ?

Reply:
Wrapped with namespace.